app/sidekiq/hca/base_submission_job.rb

```diff
@@ -16,14 +16,7 @@ def submit(user_identifier, form, google_analytics_client_id)
       begin
         result = HCA::Service.new(user_identifier).submit_form(form)
       rescue VALIDATION_ERROR
-        PersonalInformationLog.create!(data: { form: }, error_class: VALIDATION_ERROR.to_s)
```

Comment:
This logic was pulled into a private method below, but the logic inside has not changed besides adding the new StatsD metric.